src/main/resources/application.properties

```
@@ -1 +1,9 @@
+spring.datasource.url=jdbc:mysql://localhost:3306/rick_and_morty?serverTimezone=UTC
+spring.datasource.username=user
+spring.datasource.password=password
+spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
 
+spring.jpa.hibernate.ddl-auto=create-drop
+spring.jpa.show-sql=true
+
+rick-and-morty.get.characters.url=https://rickandmortyapi.com/api/character
```

```suggestion
rickAndMorty.characters.url=https://rickandmortyapi.com/api/character
```
at least `get` is odd. `rickAndMorty` or `rick-and-morty` or `rick.and.morty` is up to you

---

fixed